includes/framework/QDbBackedSessionHandler.class.php

```diff
@@ -123,6 +123,15 @@ public static function SessionRead($id) {
 			if (!$result_row) // either the data was empty or the row was not found
 				return '';
 			$strData = $result_row['data'];
+
+			if(strstr($objDatabase->Adapter, 'PostgreSql')) {
```

Comment:
@spekary I am saying the same thing. May be you misunderstood me (or I misunderstood you somewhere). For now, we have to let this one go. 

@olegabr I am going to create a new Issue right away.